commons/src/test/java/io/hyscale/commons/io/HyscaleFilesUtilTests.java

```diff
@@ -119,16 +121,17 @@ public void getFileNameTest() throws HyscaleException {
         assertEquals(SAMPLE_FILE_NAME, HyscaleFilesUtil.getFileName(sampleFilePath));
     }
     
+    
     @Test
-    public void searchFileInDirTest() {
+    public void ListFileInDirTest() {
         File file = createFile(testFilePath);
```

Comment:
Can we do creation in @BeforeAll ?

Reply:
If we do creation in @BeforeAll it will affect other tests, its better to keep it in the test itself on use case basis.

Reply:
Why other tests are effected ?